### PR TITLE
2 0 0 beta2

### PR DESCRIPTION
**Pull request by dmccraw**
Needed to update default url.
Needed to use the url passed in or default url when making the initial `TeamSnap.run`

### CODE REVIEW

File: lib/teamsnap.rb [hunk 1]
```diff
@@ -34,7 +34,7 @@ def init(opts = {})
       self.root_client = TeamSnap::Client.new(:token => token)
 
       ##   Make the apiv3 root call. collection is parsed JSON
-      collection = TeamSnap.run(root_client, :get, "/", {})
+      collection = TeamSnap.run(root_client, :get, self.url, {})
```
Comment (jjmocko) [inline lib/teamsnap.rb]: This should stay as "/".. The line is meant to say hit the root of the url you provided to the init. (the domain is set in the client)

Comment (dmccraw) [inline lib/teamsnap.rb]: The problem is that in production root is actually "/v3/"

Comment (dmccraw) [inline lib/teamsnap.rb]: If you try to hit `https://api.teamsnap.com` it gives you an error.
You actually have to hit `https://api.teamsnap.com/v3/` to get the root.

**Comment by PositiveControl** [comment]
:sheep: it

File: lib/teamsnap.rb (hunk 1 above)
Comment (ryanwi) [inline lib/teamsnap.rb]: does `Faraday.new` work with the `/v3/` path in the url? seemed like it used to have a problem with anything beyond the domain name.

Comment (dmccraw) [inline lib/teamsnap.rb]: It seems to be working with the `/v3` but I will double check. What kind of error where you seeing?

**Pull request merged**